src/frontend/ExtensionManager/components/ExtensionContents/index.tsx

```diff
@@ -24,47 +25,40 @@ const animation = {
   transition: { duration: 0.2 }
 }
 
-export function ExtensionPopup() {
+export function ExtensionPopup({ state }: { state: ExtensionState }) {
```

Comment:
since this uses the mobx state, may want to wrap this in observer too even though extensionId won't change

Reply:
you're right, this is working because the OverlayManager is wrapped in an observer, but this component should be wrapped regardless.